section-07-ci-and-publishing/model-package/requirements/test_requirements.txt

```diff
@@ -0,0 +1,13 @@
+-r requirements.txt
+
+# testing requirements
+pytest>=6.2.3,<6.3.0
+
+# repo maintenance tooling
+black==20.8b1
+flake8>=3.9.0,<3.10.0
+mypy==0.812
+isort==5.8.0
+
+# new in section 7: for fetching data in CI
+kaggle>=1.5.12,<1.6.0
```

Comment:
kaggle cli to fetch data in ci

Reply:
Are students not making the dataset part of the package? I can see that train and test csv are in manifest in. If this is the case, why do we need to fetch the dataset from kaggle? what is the use of having train and test in the index? Can they not train from the packaged data?

Reply:
@solegalli they need the data to run the package tests (since the package doesn't install itself). They don't need it to run the api tests (because that has access to the package). The only alternative would be to commit the test data, and I think the kaggle cli command can be referenced as an example for other real-life use cases (like downloading train data from S3)